### ISSUE TITLE
13 P 104716282 Film/Film number (Stuart)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#1066]
Priority (0 2 3 8) 13
Issue reported by **vk3dwu** at 2018-04-26 23:51:22 UTC
Time: 2018-04-26T23:37:03+00:00
Session ID: 1fdc89956dc63415da5a986c455ba5b4
Problem Page URL: [/manage_resources/new](/manage_resources/new)
Previous Page URL: [https://www.freereg.org.uk/cms/refinery/users/login](https://www.freereg.org.uk/cms/refinery/users/login)
Reported Issue:
In WInREG we have columns headed Image & Fiche/Film number.  However, FreeREG displays these as Film number and Film respectively.  I respectfully suggest these field descriptions should be changed in FreeREG to comply with the WinREG column headings.    

![Screenshot](http://www.freereg.org.uk/uploads/feedback/screenshot/5ae265fbf493fd0aa0227427/_old_)

**Comment by SteveBiggs** [comment]
There is ambiguity in these historical field names so we should decide if a change is appropriate.

**Comment by Sherlock21** [comment]
Well..

Not all images are on fiche. - not even if one counts film rolls loosely as fiche.

All records which I am transcribing at the moment are direct from the original PR, or from my filming thereof. So "fiche" really is not the right title. 
But I agree the field titles in FR and WinReg should be the same.

**Comment by SteveBiggs** [comment]
Perhaps they should be 'Image Group' and 'Image'? This would align them with terminology used on the image server for sources.

**Comment by Sherlock21** [comment]
Yes. So that title is wide enough for the user to put anything in there: even to the extent of: "My Own" or "Original PR" - if the images are not going to be on FR's server.

**Comment by Sherlock21** [comment]
Steve - so can you arrange to get the name done please?

**Comment by Captainkirkdawson** [comment]
Some comments.
These 2 fields have existed as they are for 10 years.
These fields were added into WinREG with some idea that maybe at some time in the future we would be able to use them to perhaps display the image. (Note the qualifiers in this statement)
FR1 had no ability to do anything with them.
FR2 simply displays the fields.
There is and can be no ability to display the original image as the location of the image is unknown.
These fields may/may not be useful for a transcriber/coordinator to locate an original image if they still have access to it. (Images provided by the LDS to us were provided under the condition we would not copy them and we would return them and a copy of the transcriptions; so displaying them would be in contravention of our agreement with LDS at the time)
As part of the image server work we have implemented the ability to display a image associated with an entry. This uses a new field that identifies the specific image on the image server. It makes no use of the prior film and film number fields.

**Comment by edickens** [comment]
A bit more background.  LDS asked for these Film and Image numbers to be added at a time when they were giving us access to film to transcribe and we were sending them the files for their use.  They needed these fields to link to the images should there be a query.  So these fields are nothing to do with linking to our image server, although, as Kirk says, if we still have the images, then this is a useful reference.
Linking to our image server's images is something completely different.

**Comment by SteveBiggs** [comment]
Thanks for the history. I am not suggesting they should link directly to the image server images, but in the case of my counties of Surrey and Sussex, in the majority of batches uploaded since I've been coordinator, I have had my transcribers enter the IS group name in the Film/Fiche number field and the image number (or part of) in the Image field. I find these invaluable when I'm checking reported transcription errors because it takes me straight to the image on the IS for that entry. I fully understand though that most other coords have not done this.

**Comment by Captainkirkdawson** [comment]
Based on Steves comment and Ericbs  I see no requirement for any change. 

If these field are used by the transcriber/coordinator then they know what they mean. If not its irrelevant.

What MIGHT be useful is to write a task that takes the 2 fields and determines if these images exist on the image server and then enter the appropriate information for the image_server_image file name for the entry

**Comment by edickens** [comment]
Perhaps, even in the very long term, WinFreeREG could access the assigned images to display them for transcribing, and at the same time enter the path to the image.

**Comment by Sherlock21** [comment]
The point of my comment was based on this is the clue as to where the source of the Transcription came from so that in the event of a query that the Transcription may be in error, the person attending to that user report would know where to go to fund the source material and evaluate the report - i.e. is there a Transcribing error ( and correct it) or not.
Nothing more, nothing less.

I raised my comments  now rather than 10 years ago, as it seemed likely that either the FR field titles or the Winreg titles should be changed in the interests of standardisation - i.e. the same thing has the same name in all the components of FR

**Comment by SteveBiggs** [comment]
I think the issue that Stuart, the originator was raising was the mapping between WinREG and FreeReg field names, i.e:

WinREG                     ->  FreeReg
Fiche/Film number    ->  Film
Image                        ->  Film number 

So there is confusion over what 'Film number' means and I have had occasions where my transcribers have reversed these fields. However, since WinREG is going obsolete soon, perhaps the confusion will no longer be an issue.

**Comment by SteveBiggs** [comment]
The issue is exactly the same with WinFreeREG since it uses the same terminolgy as WinREG.

**Comment by richpomfret** [comment]
Let's therefore update WinFreeReg and WinReg to reflect the naming conventions used in the system.

**Comment by SteveBiggs** [comment]
WinREG will not be touched anymore, but we can align WinFreeREG to the FR website names.

**Comment by stsccfr** [comment]
Closing as no longer actionable, as WinREG is not being developed and the names of the fields cannot be changed without changing them in all CSV files in which they are currently used.
